compiler/qsc_frontend/src/compile.rs

```diff
@@ -39,9 +39,9 @@ use thiserror::Error;
 bitflags! {
     #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
     pub struct RuntimeCapabilityFlags: u32 {
-        const ConditionalForwardBranching = 0b0000_0001;
```

Comment:
So, they aren't used at all at this time?

Reply:
Not yet in main but I'm using them in this [branch](https://github.com/microsoft/qsharp/tree/cesarzc/rca) from which I'll open a PR sometime soon.